src/util/resolveJsonOrRejectWithError.js

```diff
@@ -26,6 +26,10 @@ export function resolveJsonOrRejectWithError(res, options = { taxonomy: false, i
       return resolve({});
     }
 
+    if (res.status === 404) {
```

Comment:
Ser ut som det fungerer bra for å løse akkurat denne issuen, men jeg tror dette forhindrer at koden under blir kjørt. En option ignore404 (tilsvarende ignore403) kan nok løse dette.

Eventuelt se nærmere hvorfor koden under får det til å krasje.

Reply:
Pushet endring som unngikk å gjøre endringer  i denne filen